types/dynamodb-lock-client/index.d.ts

```diff
@@ -34,15 +35,15 @@ export interface FailOpenConfig extends GenericConfig {
     trustLocalTime?: boolean | undefined;
 }
 
-export class LockClient<PartitionTableKeyType extends string | number> {
+export class LockClient<PartitionTableKeyType extends string | number | Buffer | object> {
```

Comment:
so those objects are more like:
```suggestion
export class LockClient<PartitionTableKeyType extends string | number | Buffer | Record<string | Buffer | number>> {
```

>  If the type of id is Object, it is expected to have the following format:
{
  [config.partitionKey]: String|Buffer|Number,
  [config.sortKey]: String|Buffer|Number
}

Reply:
I mean: nearly everything is an object, here the shape for the object should be very specific